docs/sphinx_qtile.py

```diff
@@ -38,6 +38,9 @@
 qtile_class_template = Template('''
 .. autoclass:: {{ module }}.{{ class_name }}
     :members: __init__
+    {% if is_widget %}
+    Supported bar orientations: {{ obj.orientations }}
```

Comment:
It prints the strings set at https://github.com/kynikos/qtile/blob/develop%23573/libqtile/widget/base.py#L72 thanks to https://github.com/kynikos/qtile/blob/develop%23573/libqtile/widget/base.py#L59
I took inspiration from https://github.com/kynikos/qtile/blob/develop%23573/libqtile/bar.py#L120 but if you think it's too obscure I can change it to something else, maybe using a dictionary to translate the integers? Or I could expand the comments too.